electron/webpack/paths.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import fs from 'fs';
+
+const electronPath = fs.realpathSync(process.cwd());
+export const resolveElectron = (relativePath) => path.resolve(electronPath, relativePath);
+
+const appPath = path.resolve(electronPath, '..');
+export const resolveApp = (relativePath) => path.resolve(appPath, relativePath);
+
+const paths = {
+  electronPath,
+  electronStatic: resolveElectron('./static'),
+  electronMain: resolveElectron('./src/main/main.ts'),
+  electronPreloadJs: resolveElectron('./src/preload/preload.ts'),
+  electronAppHtml: resolveElectron('./static/app.html'),
+  electronSplashHtml: resolveElectron('./static/splash.html'),
+  electronSplashJs: resolveElectron('./src/renderer/splash.tsx'),
+  dotenv: resolveApp('.env'),
+  appPath: resolveApp('.'),
+  appSrc: resolveApp('./src'),
+  appIndexJs: resolveApp('./src/index'),
+  appPublic: resolveApp('./public'),
+};
```

Comment:
A number of these aren't used. Do we need to define them all?

Reply:
Just a copied from the guts of `create-react-app` to help define paths - may need them for webpack config later.